contrib_tools/prepare

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
```

Comment:
It would be good if these contrib tools could be split into a separate PR, as there's rather a lot going on here.